.github/workflows/main.yml

```diff
@@ -0,0 +1,49 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run.
+# Triggers the workflow on push or pull request events
+on: [push, workflow_dispatch]
```

Comment:
do we also want to have nightly tests?

Reply:
agreed to do this in the future. added in the cromshell sow